src/handlers/opensea-api.ts

```diff
@@ -6,38 +6,53 @@ import {
   NFT_API_URL,
 } from 'react-native-dotenv';
 import { rainbowFetch } from '../rainbow-fetch';
-import NetworkTypes from '@rainbow-me/networkTypes';
-import { parseAccountUniqueTokens } from '@rainbow-me/parsers';
+import NetworkTypes, { Network } from '@rainbow-me/networkTypes';
+import {
+  parseAccountUniqueTokens,
+  parseAccountUniqueTokensPolygon,
+} from '@rainbow-me/parsers';
 import { handleSignificantDecimals } from '@rainbow-me/utilities';
 import logger from 'logger';
 
-export const UNIQUE_TOKENS_LIMIT_PER_PAGE = 50;
-export const UNIQUE_TOKENS_LIMIT_TOTAL = 2000;
+export const UNIQUE_TOKENS_LIMIT_PER_PAGE: number = 50;
+export const UNIQUE_TOKENS_LIMIT_TOTAL: number = 2000;
 
 export const apiGetAccountUniqueTokens = async (
-  network: any,
-  address: any,
-  page: any
+  network: Network,
+  address: string,
+  page: number
 ) => {
   try {
+    const isPolygon = network === NetworkTypes.polygon;
     const networkPrefix = network === NetworkTypes.mainnet ? '' : `${network}-`;
     const offset = page * UNIQUE_TOKENS_LIMIT_PER_PAGE;
     const url = `https://${networkPrefix}${NFT_API_URL}/api/v1/assets`;
-    const data = await rainbowFetch(url, {
+    const urlV2 = `https://${NFT_API_URL}/api/v2/beta/assets`;
```

Comment:
nit and not a blocker: urlV2 does not rely on any additional params so can be defined as a const outside of this function